src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
```

Comment:
Should this be made an Abstract Class? You may think about it, and get back to me if anything is unclear.